src/components/legacy/TheFooter.vue

```diff
@@ -65,7 +65,7 @@ export default {
 							text: "HPI",
 						},
 						{
-							href: defaultDocuments.specificFiles(this.school.documentBaseDir)
+							href: defaultDocuments.specificFiles(this.school?.documentBaseDir)
```

Comment:
```suggestion
							href: defaultDocuments.specificFiles(this.school?.documentBaseDir || "default")
```
sonst funktioniert im Footer der Muster Schulen Link in ausgeloggtem Zustand nicht.

Reply:
Außerdem, wird das auch in den themed footers benötigt? evtl muss es noch an den anderen stellen nachgezogen werden

Reply:
pushed a better fix that fixes this directly in the method.